### ISSUE TITLE
mid-day.com

### ISSUE THREAD

**Issue by ghajini** [reek/anti-adblock-killer#1987]
### URL(s) where the issue occurs

http://m.mid-day.com/articles/bundle-of-joy-arrives-its-a-girl-for-shahid-kapoor-mira-rajput/17563182
### Describe the issue
- go to m.mid-day.com on firefox android with ublock origin
- click any news
- on going to site following message displayed
  " Blocked
  Hi, it seems there is some blocker enabled that is hampering the content to display.kindly disable or remove it to view the content "
### Screenshot(s)

![screenshot_2016-08-26-23-09-38](https://cloud.githubusercontent.com/assets/20338483/18014807/483f0cae-6be2-11e6-8b79-ab64d296ba78.png)
### Versions
- Browser/version: firefox 48 for android
- uBlock Origin version: 1.8.4
### Settings
-  uBO's default settings+adblock warning removal list+antiadblock killer
### Notes

i clicked a feature in browser ''request desktop site'' to tell site to use user agent same as pc(so that it is same as ublock on pc) but site redirects to mobile version snd detects adblocker

basically issue happens when accesing mobile version of website(iam using android smartphone with ublock orign on firefox)

**Comment by mapx-** [comment]
add this filter
`@@||mid-day.com^$script`

**Comment by ghajini** [comment]
worked
can this script block other scripts also on site

**Comment by mapx-** [comment]
that script cannot block anything :) it's an exception filter (whitelisting filter)

**Comment by mapx-** [comment]
read about filters
https://adblockplus.org/en/filters

**Comment by ghajini** [comment]
@mapx-  mean iam allowing scripts to run including ad filter

**Comment by uBlock-user** [comment]
Whitelisting like that is no way of bypassing anti adblockers, the owner will only use that to this advantage and insert first party ads and ad banners through that whitelist.

Just add this - @@||mid-day.com/resources/middaymobile/js/ads.js

**Comment by reek** [comment]
Hi,<br><br>Anti-AdBlock Killer has been updated ! <a href="https://github.com/reek/anti-adblock-killer#changelog">Changelog</a><br><br>Step 1: <a href="https://raw.githubusercontent.com/reek/anti-adblock-killer/master/anti-adblock-killer.user.js">Update</a> UserScript<br>Step 2: <a href="http://reek.github.io/anti-adblock-killer/#filterlist">Update</a> FilterList<br><br>Do not forget to donate, review, share, follow and add to favorites.<br><br>See you<br>Reek

https://github.com/reek/anti-adblock-killer/
https://twitter.com/antiadbkiller
